lib/WALASupport/SILWalaInstructionVisitor.cpp

```diff
@@ -1939,6 +1939,25 @@ jobject SILWalaInstructionVisitor::visitUncheckedOwnershipConversionInst(Uncheck
 /*                   CHECKED CONVERSIONS                                       */
 /*******************************************************************************/
 
+jobject SILWalaInstructionVisitor::visitUnconditionalCheckedCastAddrInst(UnconditionalCheckedCastAddrInst *CI) {
+  SILValue SrcValue = CI->getSrc();
+  SILValue DestValue = CI->getDest();
+
+  if (Print) {
+    llvm::outs() << "\t [UnconditionalCheckedCastAddrInst]: " << CI << "\n";
+    llvm::outs() << "\t [CONVERT]: " << CI->getSourceType().getString() << " " << SrcValue.getOpaqueValue();
+    llvm::outs() << " [TO]: " << CI->getTargetType().getString() << " " << DestValue.getOpaqueValue() << "\n";
```

Comment:
should there be a "\t" at the beginning of this info line too?

Reply:
No, its meant to print on a single line it is just very long when put on a single line

Reply:
Ah I see. Approving changes.